### PR TITLE
Update count.md

### PR DESCRIPTION
**Pull request by isradin93**
1) Fixed Python List.count() method Syntax.
2) Example 2 has TypeError; Replaced with new Example.

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/lists/terms/count/count.md [hunk 1]
```diff
@@ -42,13 +40,21 @@ print(numPen)
...
-print(numPen)
-# Output: 2
+print('77 occurred', length_77, 'times in the list.') 
+print('99 occurred', length_99, 'times in the list.') 
+print('51 occurred', length_51, 'time in the list.')
+
+# Output: 
+# 77 occurred 2 times in the list. 
+# 99 occurred 3 times in the list. 
+# 51 occurred 1 time in the list.
```
Comment (SSwiniarski) [inline content/python/concepts/lists/terms/count/count.md:59]:
````suggestion
```

This results in the following output:

```shell
77 occurred 2 times in the list. 
99 occurred 3 times in the list. 
51 occurred 1 time in the list.
````

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@isradin93, I did a first review and had one comment. We generally prefer output from examples in a separate block.

**Review by SSwiniarski — APPROVED** [review]
@isradin93 in the interest of getting this moving after a few weeks I implemented my suggestion and I'm passing this off to a second review.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@isradin93 I added a second review. Let me know if you have questions about making the changes!

**Comment by KTom101** [comment]
@isradin93 Closing this due to unresponsiveness, a new PR was made to fix the error.

**Pull request closed without merging**